content/en/legacy/flux/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Flux documentation
```

Comment:
```suggestion
title: Flux v1
```

Reply:
I used  the term `Flux Legacy` in a couple of places - WDYT about this?

Reply:
Sure let's use that instead of v1